apps/juxtaposition-ui/src/services/juxt-web/routes/console/messages.tsx

```diff
@@ -29,28 +29,30 @@ messagesRouter.get('/', async function (rawReq, res) {
 	});
 });
 
-messagesRouter.post('/new', async function (rawReq, res) {
-	const req = getAuthedRequest(rawReq);
+messagesRouter.post('/new', async function (req, res) {
+	// TODO add body validation
+	const { auth } = parseReq(req);
+	const authCtx = auth();
```

Comment:
This is probably nicer and should be done everywhere rather than inlining the auth call